### DISCUSSION TITLE
ArgoCD SyncWindow Question

### DISCUSSION THREAD

**Discussion by Mais316** [argoproj/argo-cd#10492] [Q&A]
I was wondering if the below sync window in docs, will mean to allow sync at all time or we have to make duration `24h`, in other word how to have a sync window to allow sync at all times

```sh
ID  STATUS    KIND   SCHEDULE    DURATION  APPLICATIONS  NAMESPACES  CLUSTERS  MANUALSYNC
0   Active    allow  * * * * *   1h        -             -           prod1     Disabled
```

**Comment by ishitasequeira** [comment]
The above example should allow sync at all times.
